widgets/Window.lua

```diff
@@ -48,8 +50,6 @@ function StdUi:Dialog(title, message, dialogId)
 		window = self.dialogs[dialogId];
 	else
 		window = self:Window(nil, self.config.dialog.width, self.config.dialog.height, title);
-		window:SetPoint('CENTER');
-		window:SetFrameStrata('DIALOG');
```

Comment:
No longer need to apply these new default values to the StdUi:Window

Reply:
Window and dialog is two different things. This is a big no-no.